src/datasource.rs

```diff
@@ -274,7 +280,24 @@ impl ExecutionPlan for BigtableExec {
             ..ReadRowsRequest::default()
         };
 
-        match bigtable.read_rows(request).await {
+        let future = bigtable.read_rows(request);
+        let read_rows_result = match tokio::runtime::Handle::try_current() {
+            Ok(handle) => {
+                handle.block_on(future)
```

Comment:
`cargo test` hit error of "Cannot start a runtime from within a runtime"

---

originally, `execute` was async, `bigtable.read_rows` is `await` inside `execute` and return a `MemoryStream`. I didn't find ways to continue this approach after `execute` becomes sync.

I am now thinking 2nd approach, which implements a `BigtableStream` (which can async do `bigtable.read_rows`)